pkg/trivyk8s/trivyk8s.go

```diff
@@ -161,8 +161,89 @@ func (c client) GetIncludeKinds() []string {
 	return c.includeKinds
 }
 
+// initResourceList collects scannable resources.
+func (c *client) initResourceList() {
+	// skip if resources are already created
+	if len(c.resources) > 0 {
+		return
+	}
+
+	// collect only included kinds
+	if len(c.includeKinds) != 0 {
+		// `includeKinds` are already low cased.
```

Comment:
What do you mean by this comment?

Reply:
the comment was improved https://github.com/aquasecurity/trivy-kubernetes/pull/395/commits/eed731db3655c9b849890a27e117ab687c1f2ef9